### ISSUE TITLE
Avoiding saveAsElasticsearch() reshuffle

### ISSUE THREAD

**Issue by mdvorsky** [spotify/scio#1938]
saveAsElasticsearch() currently reshuffles all data into a fixed number of shards. What do you think about adding an option to avoid AssignToShard transform? For our usecase, reshuffle is adding unnecessary overhead. (We're limiting the concurrent access to ES by --maxNumWorkers which is a lot more efficient of limiting parallelism than shuffling a lot of data into small number of very hot keys).

**Comment by nevillelyh** [comment]
@regadas WDYT? @mdvorsky if you think it's a trivial change, mind submitting a PR and we can discuss there?

**Comment by regadas** [comment]
Yeah we can definitely disable sharding! This will require adding config to allow config of batch size.

**Comment by regadas** [comment]
@alexclare will take a stab on this one. Thanks!

**Comment by RustedBones** [comment]
@regadas isn't that resolved too ?

**Comment by kellen** [comment]
Calling this done in #3566
